.travis.yml

```diff
@@ -4,4 +4,4 @@ node_js:
 before_script:
   - npm install grunt-cli -g
   - npm install
-script: grunt test
+script: npm run test
```

Comment:
please follow the conventions of [npm scripts](https://docs.npmjs.com/misc/scripts)